### ISSUE TITLE
version 1.2.0: Unknown characters in most of the console prints

### ISSUE THREAD

**Issue by bnb4505** [pavelliavonau/cmakeconverter#93]
This sequence of characters appears inside many of the strings printed to the console, even doing
cmake-converter -help has them in it. Here is the sequence.
[36m[1m

I followed the instructions found on the documentation page for installing and running the converter.

**Comment by pavelliavonau** [comment]
Describe conditions that you are working in, please.

**Comment by bnb4505** [comment]
Im working on windows 64 bit, running it as admin command line. Im trying to build a very large c++ framework that has one overlying .sln file. Im using python 3.7, and pip for the respective version. Running cmake-converter with an incorrect flag, such as "cmake-converter -help" outputs this:
``ERR  : [31m[1mCommand line parsing error:
Usage:
        cmake-converter [-h]
        cmake-converter [-V]
        cmake-converter (-s solution | -p project) [-i]
                        [-c cmake]
                        [-D dependency...]
                        [-O cmakeoutput]
                        [-a codefile]
                        [-I cmakefile]
                        [-S std].[39m[0m
~~~~~~~~~~~~~~~~~~~~~~~~~~
Exiting with error code: 64``

When I run the program, it successfully creates .txt files, but the output to console is full of these 
``[39m[0m`` characters

**Comment by pavelliavonau** [comment]
This issue and many others also fixed at develop. Try it then. I shall not going to fix release 1.2.
Also you may try pre release version. It works much better then 1.2.

**Comment by pavelliavonau** [comment]
Closes due new stable release.
